chain/x/zoracle/internal/types/request.go

```diff
@@ -4,6 +4,14 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
+type ResolveStatus int
+
+const (
+	Pending ResolveStatus = iota
```

Comment:
Let's call them `Open`, `Success`, `Failure`. Pending is a bit confusing with pending list.